cmd/ctr/list.go

```diff
@@ -19,21 +19,101 @@ var containersCommand = cli.Command{
 var listCommand = cli.Command{
 	Name:    "list",
 	Aliases: []string{"ls"},
-	Usage:   "list containers",
+	Usage:   "list containers & tasks",
+	Subcommands: cli.Commands{
+		taskList,
+		containerList,
+	},
+	Action: taskListFn,
```

Comment:
would name it `ListFn` and made it so that it invokes both `containerList` and `taskList`